Library/Koala/Koala.swift

```diff
@@ -494,6 +507,12 @@ public final class Koala {
       ])
   }
 
+  public func trackSaveProject(context: SaveContext) {
+    self.track(event: "Star", properties: [ // check this
```

Comment:
Thanks! this is not being used we are now tracking with `trackProjectSave(_ project: Project, context: SaveContext)`.